packages/next-plugin-material-ui/src/enhance-app-server.js

```diff
@@ -0,0 +1,6 @@
+import { ServerStyleSheets } from '@material-ui/styles'
+
+export default async function enhanceApp (ctx) {
```

Comment:
This hook should probably be unstable, right?

Reply:
Updated